R/derive_interp_records.R

```diff
@@ -24,7 +24,8 @@
 #'        `parameter = "HEIGHT"`,
 #'   or   `parameter = "BMI"`.
 #'
-#' @return The input dataset plus additional interpolated records.
+#' @return The input dataset plus additional interpolated records: a record
+#'         for each day from the minimum age to the maximum age.
```

Comment:
It should be mentioned how the non-expected variables are populated in the output dataset.